package.json

```diff
@@ -59,13 +59,14 @@
     "@types/robust-point-in-polygon": "^1.0.2",
     "@vitejs/plugin-react": "^1.3.2",
     "autoprefixer": "^10.4.7",
-    "clsx": "^1.2.0",
-    "eslint": "^8.18.0",
-    "eslint-config-cheminfo-react": "^9.0.2",
-    "eslint-config-cheminfo-typescript": "^11.0.1",
-    "immer": "^9.0.15",
-    "jest": "^28.1.2",
-    "jest-image-snapshot": "^5.1.0",
+    "clsx": "^1.1.1",
+    "eslint": "^8.16.0",
+    "eslint-config-cheminfo-react": "^8.2.1",
+    "eslint-config-cheminfo-typescript": "^10.4.0",
+    "immer": "^9.0.14",
+    "jest": "^28.1.0",
+    "jest-image-snapshot": "^4.5.1",
+    "jest-matcher-deep-close-to": "^3.0.2",
```

Comment:
I think there was a mistake when fixing a conflict. Can you put back the latest versions of packages from `clsx` to `jest-image-snapshot`?
This is probably the cause of the error you had on the import of `clsx` a few days ago.